lib/internal/http2.js

```diff
@@ -608,9 +601,7 @@ function incomingOnRead(nread, buffer) {
       }
     }
     return;
-  }
-
-  if (nread === 0) {
+  } else if if (nread === 0) {
```

Comment:
Sorry. I had to tweet this.
https://twitter.com/mightyiam/status/805822151462895616
Thing is, I don't know, either. Is it valid?